ush/config.sh.gefs02

```diff
@@ -34,7 +39,7 @@ NUM_ENS_MEMBERS="1"
 NDIGITS_ENSMEM_NAMES="2"
 
 # Get external model files staged on disk
-COMINgfs="/lustre/data/gefs"
+COMINgfs="/lustre/FFAIR/data/gefs"
```

Comment:
This is not a problem, but we may have lost the ability to run cases with staged data when I put in the download script. 😬